symint attributes

avikchaudhuri opened the pull request:
Differential Revision: D78747585


cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv @voznesenskym @penguinwu @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @jiayisunx @chenyang78 @kadeng @chauhang @amjames @Lucaskabela

albanD commented on torch/nn/utils/stateless.py:
```diff
@@ -103,15 +103,17 @@ def _reparametrize_module(
     stack_weights: bool = False,
 ):
-    parameters_and_buffers = parameters_and_buffers
+    parameters_and_buffers_only = {
+        k: v for k, v in parameters_and_buffers.items() if isinstance(v, Tensor)
+    }
```
That is correct afaik

albanD reviewed (commented):
Not sure what this is about given there are no PR description?

avikchaudhuri committed:
symint attributes (#159057)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/159057

Rollback Plan:

Differential Revision: D78747585
